include/IMeshBuffer.h

```diff
@@ -625,6 +645,11 @@ GL_INVALID_OPERATION is generated by glVertexAttribPointer if size is GL_BGRA an
 		    return outPos;
         }
 
+		//! Accesses data of buffer of attribute of given id
+		/** @param attrId Attribute id.
+		@returns Pointer to corresponding buffer's data incremented by `baseVertex`
```

Comment:
and by 'bufferOffset' also

Basically it will get the start of the array at the same point as OpenGL will get upon a glDraw*